airflow-core/src/airflow/ui/public/i18n/locales/en/common.json

```diff
@@ -318,7 +318,10 @@
   "total": "Total {{state}}",
   "triggered": "Triggered",
   "tryNumber": "Try Number",
-  "user": "User",
+  "user": {
+    "signedInAs": "Signed in as",
+    "user": "User"
```

Comment:
The value from browse could be used here to avoid maintaining duplicate translation.

```
public/i18n/locales/en/browse.json
10:      "user": "User",
```